README.md

```diff
@@ -66,13 +66,19 @@ install all the dependencies.
 The application uses Postgres for the database, so you need to have that on
 your server. You need to create a database, which you can do with `createdb
 gavel` (unless you're using a different database name). Before you use the app,
-you need to initialize the database by running `python initialize.py`. **Note
+you need to initialize the database by running `python initialize.py`. **Note 
 that Gavel does not preserve database schema compatibility between versions.**
 
+In order to send emails, you'll need to have a broker, like Redis, to 
+distribute email tasks to a Celery worker. 
```

Comment:
Instead of saying that people need to install a broker like Redis, we should just tell people to install Redis (especially because the `requirements.txt` installs redis support, but doesn't install support for other brokers). Also, we don't need to explain the reason/mechanism here.

So maybe say something like "In order to send emails, you'll need to install Redis."